packages/select/custom-resources/AvOrganizationSelect.js

```diff
@@ -22,6 +27,12 @@ AvOrganizationSelect.propTypes = {
   resourceIds: PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.arrayOf(PropTypes.string),
+    PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
+  ]),
+  permissionIds: PropTypes.oneOfType([
```

Comment:
for `dx`, `permissionIds` and `resourceIds` should be able to be either strings or numbers. most apps have permissions set as constants, many of them number constants. It would be a pain to have to cast your constants to strings to avoid getting warnings using this component.